satellite/repair/repairer/segments.go

```diff
@@ -248,9 +248,19 @@ func (repairer *SegmentRepairer) Repair(ctx context.Context, path storj.Path) (s
 	// add pieces that failed piece hashes verification to the removal list
 	toRemove = append(toRemove, failedPieces...)
 
+	pointer.LastRepaired = time.Now().UTC()
+	pointer.RepairCount++
+
 	// Update the segment pointer in the metainfo
 	_, err = repairer.metainfo.UpdatePieces(ctx, path, pointer, repairedPieces, toRemove)
-	return err == nil, err
+	if err != nil {
+		return false, err
+	}
+
+	mon.IntVal("checker_segment_time_until_repair").Observe(int64(time.Since(pointer.LastRepaired).Seconds()))
```

Comment:
These metrics don't have to do with checker so I don't think `checker` should be in the name. Maybe `segment_time_until_repair` and `segment_repair_count`